benchmarks/user/benchmark.py

```diff
@@ -245,8 +246,13 @@ def run_jit_backdoor(problem, **kwargs):
 
     info("Running wave propagation Operator...")
 
-    configuration['jit-backdoor'] = True
-    solver.forward(autotune=autotune)
+    @switchconfig(jit_backdoor=True)
+    def _run_jit_backdoor():
+        return solver.forward(autotune=autotune)
+
+    retval = _run_jit_backdoor()
+
+    return retval
```

Comment:
`return _run_jit_backdoor()` ?

Reply:
fixed. I'm developing an aesthetic preference for splitting the two TBH, and I have no idea why !